yb-voyager/cmd/importData.go

```diff
@@ -1016,7 +1016,7 @@ func dropIdx(conn *pgx.Conn, idxName string) {
 
 func executeSqlFile(file string, objType string, skipFn func(string, string) bool) {
 	log.Infof("Execute SQL file %q on target %q", file, target.Host)
-
+	mergeSqlFilesIfNeeded(file, objType)
```

Comment:
We should merge files towards the end of `export schema` phase.
If you execute it at the time of `import schema`, analyze-schema won't find the definitions in the individual DDL files.